src/lib/style/Core.scss

```diff
@@ -967,3 +971,10 @@ a.dropdown-item {
 .plotly .slice {
   cursor: pointer;
 }
+
+hr ~ .SRC-wrapper {
```

Comment:
I think this scss should be put in the GenerateComponentsFromRow's [scss](https://github.com/Sage-Bionetworks/portals/blob/master/app-template/src/portal-components/GenerateComponentsFromRow.scss) instead because the hr is only added by that component

Reply:
So i hesitated with this one.
Part of me agrees with the above statement but on the other hand it doesn't know about the
.SRC-wrapper   .SRC-marginTopMinusTen  components and the margin gets applied to those components and not to hr